### ISSUE TITLE
possible encoding issue?

### ISSUE THREAD

**Issue by jpmarindiaz** [nicolewhite/RNeo4j#26]
Hi
I am having some issues with encoding.
Reproducible example here:

```
library(RNeo4j)
graph = startGraph("http://localhost:7474/db/data/")
graph$version
clear(graph)
mugshots = createNode(graph, "Bar", name = "Mugshots", location = "México")
mugshots$location
#[1] "MÃ©xico"
```
Any ideas?
My guess is that for some reason RNeo4j is not keeping UTF-8 encoding when parsing json responses

Cheers

**Comment by nicolewhite** [comment]
Thanks for the reproducible example. I'll investigate.

**Comment by nicolewhite** [comment]
This is fixed in 1.3.3. Long story short, RCurl's basicTextGatherer didn't encode anything correctly. Ended up switching to httr.

```r
> library(RNeo4j)
> graph = startGraph("http://localhost:7474/db/data/")
> graph$version
[1] "2.2.0"
> clear(graph, input=F)
> mugshots = createNode(graph, "Bar", name = "Mugshots", location = "México")
> mugshots$location
[1] "México"
```
